src/org/usfirst/frc4904/sovereignty/PIDContainerOrchestrator.java

```diff
@@ -0,0 +1,54 @@
+package org.usfirst.frc4904.sovereignty;
+
+
+import java.util.HashMap;
+import java.util.Map;
+import org.usfirst.frc4904.sovereignty.PIDContainer.PIDContainerModifier;
+
+public class PIDContainerOrchestrator {
+	private static PIDContainerOrchestrator singleton;
+	private final Map<String, PIDContainer> containers;
+
+	private PIDContainerOrchestrator() {
+		containers = new HashMap<String, PIDContainer>();
+	}
+
+	public static PIDContainerOrchestrator getInstance() {
```

Comment:
make `synchronized` so as not to inadvertently create TWO singletons (catastrophic). (Yes you can put synchronized on a static method)